packages/frontend/pages/index.tsx

```diff
@@ -112,4 +112,4 @@ const Home: NextPage = () => {
 
 export default Home
 
-const VIVO_EXPORTER_URL = process.env.NEXT_VIVO_EXPORTER_URL ?? '/vivo';
+const VIVO_EXPORTER_URL = (process.env.NEXT_PUBLIC_VIVO_EXPORTER_URL ?? process.env.NEXT_PUBLIC_VIVO_BASE_PATH ?? 'http://127.0.0.1:2025').trim()
```

Comment:
Should this have the port?

Reply:
I restored default  to `http://127.0.0.1:2025'` because it simplifies frontend development independently of the golang proxy. There's no need for port when specifying NEXT_PUBLIC_VIVO_BASE_PATH, because it will be relative to the frontend's origin.